src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -127,6 +128,36 @@ public void testKeyEncryption() throws InvalidParametersException {
         assertEquals(msg, decrptedMsg);
     }
 
+    private static String encryptWithoutSpecifyingAlgorithmParams(String value) {
+        try {
+            SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+            Cipher cipher = Cipher.getInstance("AES");
+            cipher.init(Cipher.ENCRYPT_MODE, sks, cipher.getParameters());
+            byte[] encrypted = cipher.doFinal(value.getBytes());
+            return StringHelper.byteArrayToHexString(encrypted);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+
+    }
+
+    @Test
+    public void testDefaultAesCipherParams() throws InvalidParametersException {
+        String msg1 = "AAAAAA";
+        //128 bits and more
+        String msg2 = "AAAAAAAAAABBBBBBBBBBCCCCCCCCCCCC";
```

Comment:
Would also be good to add a comment to explain why we need to test for encrypting strings > 128 bits